google/cloud/pubsub/quickstart/quickstart.cc

```diff
@@ -27,18 +27,19 @@ int main(int argc, char* argv[]) try {
 
   // Create a namespace alias to make the code easier to read.
   namespace pubsub = ::google::cloud::pubsub;
+
   auto publisher = pubsub::Publisher(
       pubsub::MakePublisherConnection(pubsub::Topic(project_id, topic_id)));
   auto id =
       publisher
           .Publish(pubsub::MessageBuilder{}.SetData("Hello World!").Build())
           .get();
   if (!id) throw std::move(id).status();
-  std::cout << "Hello World published with id=" << *id << "\n";
+  std::cout << "Hello World published with id=" << *id << std::endl;
 
   return 0;
 } catch (google::cloud::Status const& status) {
-  std::cerr << "google::cloud::Status thrown: " << status << "\n";
+  std::cerr << "google::cloud::Status thrown: " << status << std::endl;
   return 1;
 }
 //! [END pubsub_quickstart_publisher] [all]
```

Comment:
cloud.google.com probably has code referencing the region tag in this file.  See my other comments regarding moving region tags.